engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubConnection.java

```diff
@@ -136,4 +143,11 @@ private URIBuilder appendToPath(final URIBuilder uriBuilder, final String pathSe
 
         return uriBuilder.setPath(pathSegment);
     }
+
+    private class DataHubRequestRetryHandler extends DefaultHttpRequestRetryHandler {
+        DataHubRequestRetryHandler() {
+            super(3, false,
+                    Arrays.asList(InterruptedIOException.class, UnknownHostException.class, SSLException.class));
```

Comment:
BTW, any reason to retry connect exception? That's usually connection refused. I don't think timeouts would trigger that.